pages/reference-sdk-react-hooks/useconfirmtransaction.mdx

```diff
@@ -0,0 +1,116 @@
+import { Tabs } from 'nextra/components'
+
+# `useConfirmTransaction`
+
+`(params?: UseConfirmTransactionParams): UseConfirmTransactionReturnType`
+
```

Comment:
Missing a short description before jumping to the example.

Reply:
Done